internal/proxy/options.go

```diff
@@ -48,11 +48,11 @@ import (
 type Options struct {
 	Port int `envconfig:"PORT" default:"4180"`
 
-	ProviderURLString      string `envconfig:"PROVIDER_URL"`
-	ProxyProviderURLString string `envconfig:"PROXY_PROVIDER_URL"`
-	UpstreamConfigsFile    string `envconfig:"UPSTREAM_CONFIGS"`
-	Cluster                string `envconfig:"CLUSTER"`
-	Scheme                 string `envconfig:"SCHEME" default:"https"`
+	ProviderURLString         string `envconfig:"PROVIDER_URL"`
+	ProviderURLInternalString string `envconfig:"PROVIDER_URL_INTERNAL"`
```

Comment:
I'd rather not make a breaking change here in the env-var name if possible even if we do it internally to make things clearer.  if we do change things, let's make sure we update the things like the quickstart:
https://github.com/buzzfeed/sso/blob/6db5495e8d4bdd50244e94cc2369517f37f74f38/quickstart/docker-compose.yml#L32

Reply:
should we try to re-jigger the kubernetes quick start here https://github.com/buzzfeed/sso/blob/6db5495e8d4bdd50244e94cc2369517f37f74f38/quickstart/kubernetes/sso-proxy-deployment.yml#L26-L27 to enable the internal connection between sso-proxy->sso-auth ?

Reply:
yep, good idea - added.